.stylelintrc

```diff
@@ -1,10 +1,8 @@
 {
-  "extends": "stylelint-config-standard",
+  "extends": ["stylelint-config-standard", "stylelint-config-prettier"],
```

Comment:
[`stylelint-config-prettier`](https://github.com/prettier/stylelint-config-prettier) basically disables all `stylelint` rules that conflict with prettier.

Stylelint was updated from v14 to v15 within hours of these PRs being made, and it happens to be incompatible with `stylelint-config-prettier`. We're at v14 for now, but `stylelint-config-prettier` is an official package in the same repo so it likely will be updated soon.